auth/resource.cc

```diff
@@ -79,6 +79,13 @@ static permission_set applicable_permissions(const service_level_resource_view &
 }
 
 static permission_set applicable_permissions(const functions_resource_view& fv) {
+    if (fv.function_name() || fv.function_signature()) {
+        return permission_set::of<
+                permission::ALTER,
+                permission::DROP,
+                permission::AUTHORIZE,
+                permission::EXECUTE>();
+    }
     return permission_set::of<
             permission::CREATE,
```

Comment:
missing: test

Reply:
Ok, I added a test and a second one is in the auth_roles_test::TestAuthRoles::test_filter_granted_permissions_by_resource_type dtest